### ISSUE TITLE
Update react scripts and babel-loader

### ISSUE THREAD

**Issue by winkerVSbecks** [rangle/radius#70]

**Comment by t-schemmer** [comment]
Is this still a relevant issue?

**Comment by winkerVSbecks** [comment]
Yes, react-scripts in the app can be updated. However, there was an issue with eslint and the react-scripts itself. Was waiting for a fix before updating. You can test by updating and run build.
